core/src/main/java/cz/fi/muni/pv217/employee/management/entity/Order.java

```diff
@@ -0,0 +1,32 @@
+package cz.fi.muni.pv217.employee.management.entity;
+
+import cz.fi.muni.pv217.employee.management.enums.OrderState;
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+@Entity
+@Table(name = "orders")
```

Comment:
Do we need both table and entity?

Reply:
Yes, we do. `order` is probably the key word in postgres.